include/domain_bridge/domain_bridge.hpp

```diff
@@ -133,6 +135,18 @@ class DomainBridge
     const TopicBridge & topic_bridge,
     const TopicBridgeOptions & options = TopicBridgeOptions());
 
+  /// Bridge a service from one domain to another.
+  /**
+   * \param service_bridge: Struct containing info about the service to bridge.
+   * \param options: Options for bridging the topic.
```

Comment:
This docblock is outdated. We should make sure to document exactly what the `from_domain_id` and `to_domain_id` mean here (similar to the design doc).

Reply:
Thanks, updated in https://github.com/ros2/domain_bridge/pull/26/commits/dea8e9972a025ed0ec9807b4beee9734a74692ac.